pages/signup.js

```diff
@@ -613,8 +619,7 @@ export default function Signup(props) {
                   className="block leading-tight text-sm font-body mb-5 lg:text-p font-bold"
                   aria-required="true"
```

Comment:
For whatever reason this won't announce required now on the grouping, but it works if you set `aria-required="true"` on the `fieldset` instead

Reply:
`aria-required="true"` on the `fieldset` creates an a11y cypress error, but it works if we put required directly on the radio inputs.

Reply:
This isn't what we want either though since screen readers would announce required on both radio buttons, which would be confusing for screen reader users. This was pointed out to us as an example when we took a11y training and was originally addressed in #457 but that solution no longer seems to work in this PR.

Reply:
I've tried a few things to get this working to no avail. For whatever reason NVDA will no longer register that `aria-required` is present on the `legend`. A workaround for now would be to do something like below and create a bug issue:

```html                
<legend className="block leading-tight text-sm font-body mb-5 lg:text-p font-bold">
  <b className="text-error-border-red" aria-hidden="true">
     *
  </b>{" "}
  {t("formLang")}
  <span className="sr-only">required</span>
</legend>
```